packaging/opensuse/snapd.spec

```diff
@@ -341,6 +341,8 @@ fi
 %dir %{_sharedstatedir}/snapd/lib/gl32
 %dir %{_sharedstatedir}/snapd/lib/vulkan
 %dir %{_localstatedir}/cache/snapd
+# The {_systemdgeneratordir} macro seems to exist only in tumbleweed so it's not used here.
+%dir /usr/lib/systemd/system-generators
```

Comment:
`%{_prefix}/lib/systemd/...` to be consistent with other locations

or maybe something like this at the top?
```
%{!?systemdgeneratordir: %global systemdgeneratordir %{_prefix}/lib/systemd/system-generators}
```
and use `%{systemdgeneratordir}` everywhere then